monkey/monkey_island/cc/services/reporting/exploitations/monkey_exploitation.py

```diff
@@ -0,0 +1,60 @@
+import logging
+from dataclasses import dataclass
+from typing import List
+
+from monkey_island.cc.database import mongo
+from monkey_island.cc.services.node import NodeService
+from monkey_island.cc.services.reporting.issue_processing.exploit_processing.exploiter_descriptor_enum import (  # noqa: E501
+    ExploiterDescriptorEnum,
+)
+
+logger = logging.getLogger(__name__)
+
+
+@dataclass
+class MonkeyExploitation:
+    label: str
+    ip_addresses: List[str]
+    domain_name: str
+    exploits: List[str]
+
+
+def get_monkey_exploited() -> List[MonkeyExploitation]:
+    exploited_with_monkeys = [
+        NodeService.get_displayed_node_by_id(monkey["_id"], True)
+        for monkey in mongo.db.monkey.find({}, {"_id": 1})
+        if not NodeService.get_monkey_manual_run(NodeService.get_monkey_by_id(monkey["_id"]))
+    ]
+
+    exploited_without_monkeys = [
+        NodeService.get_displayed_node_by_id(node["_id"], True)
+        for node in mongo.db.node.find({"exploited": True}, {"_id": 1})
+    ]
+
```

Comment:
Maybe we can come up with better names for these lists. I'm not sure what "exploited_without_monkeys" means.